copy_dev_postgres_to_v14.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/sh -e
+target_namespace="hmpps-interventions-dev"
+
+echo "Going to stop and migrate $target_namespace"
+echo "Please verify             ^^^^^^^^^^^^^^^^^^^^^^^^^^^ and abort if necessary."
+sleep 2
+
+
+echo
+echo "✨ Stopping UI"
+kubectl scale \
+  --namespace="$target_namespace" \
+  --replicas=0 \
+  --timeout=5m \
+  deployment/hmpps-interventions-ui
+
+
+echo
+echo "✨ Waiting for all pods to fully shut down"
+kubectl wait --for=delete --timeout=-1s \
+  --namespace="$target_namespace" \
+  --selector=release=hmpps-interventions-ui \
+  pod
+
+
+echo
+echo "✨ Stopping API/jobs"
+kubectl scale \
+  --namespace="$target_namespace" \
+  --replicas=0 \
+  --timeout=5m \
+  --selector=app.kubernetes.io/name=hmpps-interventions-service \
```

Comment:
"Scale to 0" on the service/API/batch jobs level, ensuring we don't have an active database connection.

Note: this won't prevent scheduled jobs from kicking off.